tutorials-and-examples/inference-servers/jetstream/pytorch/jetstream-pytorch-server/Dockerfile

```diff
@@ -4,8 +4,7 @@
 FROM ubuntu:22.04
 
 ENV DEBIAN_FRONTEND=noninteractive
-ENV PYTORCH_JETSTREAM_VERSION=jetstream-v0.2.0
-ENV JETSTREAM_VERSION=v0.2.0
+ENV PYTORCH_JETSTREAM_VERSION=v0.2.2
```

Comment:
this is the latest stable branch, any breaking changes to the existing functionalities?

Reply:
no breaking changes

Reply:
Jetstream latest release is v0.2.1, why is pytorch on a later release and not in sync with the Jetstream?

Reply:
We need a particular commit: https://github.com/google/JetStream/pull/84

Reply:
Thanks, has this dockerfile been built and tested on GKE yet?

Reply:
Looks like https://github.com/GoogleCloudPlatform/ai-on-gke/commit/3096a0e2e5fcf333bee4f23f2a435f65b4de0412#diff-32a8afd80b59d025388e8c0291592744fa2819f928908d65d62cc85518234fe0 was submitted and accounts for this change. Could we close this PR then?